tool/src/org/antlr/v4/codegen/target/JavaScriptTarget.java

```diff
@@ -94,6 +94,6 @@ public boolean supportsOverloadedMethods() {
 
 	@Override
 	public boolean isATNSerializedAsInts() {
```

Comment:
The method should be removed since the basic implementation is the same.

Reply:
I'd suggest making the overridden method purely abstract. The base class shouldn't make any assumption about the derived classes.

Reply:
Not sure since we only need strings for Java runtime. Other runtimes can use ints for encoding.

Reply:
When I/we can shift others to int-only then I guess we can remove method. good idea.